fe/fe-core/src/main/java/org/apache/doris/common/profile/ExecutionProfile.java

```diff
@@ -243,6 +247,81 @@ public void update(long startTime, boolean isFinished) {
         }
     }
 
+    public Status updateProfile(TQueryProfile profile, TNetworkAddress address) {
+        if (isPipelineXProfile) {
+            if (!profile.isSetFragmentIdToProfile()) {
+                return new Status(TStatusCode.INVALID_ARGUMENT, "FragmentIdToProfile is not set");
+            }
+
+            for (Entry<Integer, List<TDetailedReportParams>> entry : profile.getFragmentIdToProfile().entrySet()) {
+                int fragmentId = entry.getKey();
+                List<TDetailedReportParams> fragmentProfile = entry.getValue();
+                int pipelineIdx = 0;
+                List<RuntimeProfile> taskProfile = Lists.newArrayList();
+                for (TDetailedReportParams pipelineProfile : fragmentProfile) {
+                    String name = "Pipeline :" + pipelineIdx + " "
+                            + " (host=" + address + ")";
```

Comment:
the address is wrong , please refer the new updateProfile code to modify it.

Reply:
done